templates/tab/js/non-sso/src/components/sample/AddSSO.jsx

```diff
@@ -4,7 +4,7 @@ export function AddSSO(props) {
   const { ssoDocUrl, fluentDocUrl, readmeUrl } = {
     ssoDocUrl: "https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/auth-aad-sso?tabs=dotnet",
     fluentDocUrl: "https://developer.microsoft.com/en-us/fluentui#/get-started/web",
-    readmeUrl: "https://aka.ms/teamsfx-non-sso-tab-sample-readme", 
+    readmeUrl: "https://aka.ms/teamsfx-add-sso", 
```

Comment:
Just curious why there is https but not in README. Make it consistent?

Reply:
Good catch. Updated in readme, thx！